FindOnPage.md

```diff
@@ -0,0 +1,664 @@
+        /// typically provides a default handling approach, while the document's language settings (e.g., specified using the HTML lang attribute) can override these defaults to apply locale-specific rules. This dual consideration 
+        /// ensures that text is processed in a manner consistent with user expectations and the linguistic context of the content.
+        Boolean IsCaseSensitive { get; set; };
+
+        /// Similar to case sensitivity, word matching also can vary by locale, which may be influenced by both the browser's UI locale and the document's language settings. The browser's UI locale         
+        /// typically provides a default handling approach, while the document's language settings (e.g., specified using the HTML lang attribute) can override these defaults to apply locale-specific rules. This dual consideration 
+        /// ensures that text is processed in a manner consistent with user expectations and the linguistic context of the content.
+        /// ShouldMatchWord determines if only whole words should be matched during the find operation. Returns TRUE if only whole words should be matched, FALSE otherwise.
+        Boolean ShouldMatchWord { get; set; };
+        
+        /// Gets or sets the state of whether all matches are highlighted. 
+        /// Returns TRUE if all matches are highlighted, FALSE otherwise.
+        /// Note: Changes to this property take effect only when StartFind, FindNext, or FindPrevious is called. 
+        /// Preferences for the session cannot be updated unless another call to the StartFind function on the server-side is made. 
+        /// Therefore, changes will not take effect until one of these functions is called.
+        Boolean ShouldHighlightAllMatches { get; set; };
+        
+        /// Sets this property to hide the default Find UI. 
+        /// You can use this to hide the default UI so that you can show your own custom UI or programmatically interact with the Find API while showing no Find UI. 
+        /// Returns TRUE if hiding the default Find UI and FALSE if using showing the default Find UI.
+        /// Note: Changes to this property take effect only when StartFind, FindNext, or FindPrevious is called. 
+        /// Preferences for the session cannot be updated unless another call to the StartFind function on the server-side is made. 
+        /// Therefore, changes will not take effect until one of these functions is called.
+        Boolean SuppressDefaultFindDialog { get; set; };
+    };
+
+   runtimeclass CoreWebView2Find : [default]ICoreWebView2Find {}
+
+    /// Interface providing methods and properties for finding and navigating through text in the web view.
+    /// This interface allows for finding text, navigation between matches, and customization of the find UI.
+    [com_interface("staging=ICoreWebView2StagingFind")]
+    [ms_owner("core", "maxwellmyers@microsoft.com")]
+    [availability("staging")]
+    interface ICoreWebView2Find 
```

Comment:
If I call WebView.Find to get the Find object, there is no way for me to query what state it is in. If I want to know that, I have to keep track myself.

For example:
Is there an active Find operation in place at the moment? I can figure this out by checking `ActiveMatchIndex==-1` but that is a little obscure.
If there is an active Find operation in place, what string is being searched for? There is no way for me to get to the FindConfiguration object.